app/domain/errors.rb

```diff
@@ -575,17 +575,17 @@ module AuthnJwt
       )
 
       InvalidPublicKeys = ::Util::TrackableErrorClass.new(
-        msg: "Failed to parse 'public-keys': {0-parse-error}",
+        msg:  "Failed to parse 'public-keys': {0-parse-error}",
         code: "CONJ00120E"
       )
 
       InvalidSigningKeyType = ::Util::TrackableErrorClass.new(
-        msg: "Signing key type '{0-type}' is invalid",
+        msg:  "Signing key type '{0-type}' is invalid",
         code: "CONJ00121E"
       )
 
       InvalidSigningKeySettings = ::Util::TrackableErrorClass.new(
-        msg: "Invalid signing key settings: {0-validation-error}",
+        msg:  "Invalid signing key settings: {0-validation-error}",
```

Comment:
what is the difference from this one to CONJ00121E ?

Reply:
CONJ00122E - is a validation time error. Will appear each time when configuration is invalid.
CONJ00121E - is a runtime error of the `CreateSigningKeyProvider` factory class. It exists in order to make the factory [switch case statement](https://github.com/cyberark/conjur/blob/3965075b5ff82da63cee6e0b3ba6fc5eecf09a9b/app/domain/authentication/authn_jwt/signing_key/create_signing_key_provider.rb#L50) fully deterministic. I guess that a customer will never see CONJ00121E error.

Reply:
fine from English point of view